create-event/src/components/EventCreator.jsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react'
+import moment from 'moment'
+
+// containers
+import Ticket from '../containers/Ticket'
+import ImgUpload from '../containers/ImgUpload'
+import Description from '../containers/Description'
+import Venue from '../containers/Venue'
+
+// Material UI
+import Paper from 'material-ui/Paper'
+import TimePicker from 'material-ui/TimePicker'
+import DatePicker from 'material-ui/DatePicker'
+
+// Filestack API - PICATIC
+// const FILESTACK_API_KEY = 'An8cT0rwoR8O7aDBiPJHrz'
+
+// Filestack API - PERSONAL
+const FILESTACK_API_KEY = 'AwQ1CrYCYTo6HCuVyPbTBz'
+
+const EventCreator = props => {
+  const {
+    event,
+    tickets,
+    submitted,
+    handleEventChange,
+    handleEventName,
+    handleTimeChange,
+    handleTicketChange,
+    handleDescriptionChange,
+    addTicket,
+    deleteTicket,
+    updateEvent,
+    activateEvent
+  } = props
+
+  if (!event) {
+    return false
+  }
+
+  const hasEvent = !isNaN(event.id)
+
+  const {
+    title,
+    start_date,
+    start_time,
+    end_time,
+    cover_image_uri
+  } = event.attributes
+
+  const startDate = start_date
+    ? new Date(moment(start_date, 'YYYY-MM-DD').toISOString())
+    : null
+  const startTime = start_time
+    ? new Date(moment(start_time, 'HH:mm:ss').toISOString())
+    : null
+  const endTime = end_time
+    ? new Date(moment(end_time, 'HH:mm:ss').toISOString())
+    : null
+
+  // FIXME: How can we make this logic simpler
```

Comment:
I would consider making the jsx here into a seperate presentational component.

Something like:
`const renderTickets = hasTickets && <TicketHeader />`